### DISCUSSION TITLE
Ja4+ and Zeek Parsing

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#13880] [Unsupported Versions]
Hello everyone!

I'm looking to incorporate ja4+ fingerprinting and need a little advice on how to parse those specific fields. What would be the best way to extract ja4 fields from message much like how ja3 is being parsed as hash.ja3? Thanks in advance!

![image](https://github.com/user-attachments/assets/70f8d570-44ea-4a2d-9242-5d2e7aa9f09b)
![image](https://github.com/user-attachments/assets/854b3846-629c-4146-be9f-9718394c6db6)

**Comment by ColeVan** [comment]
After reviewing the zeek.ssl pipeline configuration I was able to figure out how to parse ja4 and ja4s from the message field. I'm using zeek ja4 configurations from this github link below if anyone else is interested.
https://github.com/FoxIO-LLC/ja4/tree/main/zeek
![image](https://github.com/user-attachments/assets/11cb1649-9230-42c6-8a55-0bc760c89522)
![image](https://github.com/user-attachments/assets/e1d63d0e-c983-4df2-bbb9-7489c6c7bbc8)

**Comment by reyesj2** [comment]
Noticed there wasn't an issue to keep this from being lost. Created #14465 to investigate including this into a future release.
